rotkehlchen/externalapis/etherscan.py

```diff
@@ -221,7 +221,7 @@ def _query(
         result = None
         api_key = self._get_api_key()
         if api_key is None:
-            if not self.warning_given:
+            if self.chain == SupportedBlockchain.ETHEREUM and not self.warning_given:
```

Comment:
Hmmm from change-wise this is the fastest I guess, but can you perhaps not move this into the etherscan superclass only and not have an extra if check instead for every single etherscan query?

I mean in the end the whole thing needs to go away and have 1 class so then perhaps you can have an even simpler hack? Turn the `self.warning_given` into a file level variable and set it once? Though that's super super hacky